regulations/templates/regulations/preamble-partial.html

```diff
@@ -17,12 +17,11 @@
             <div class="queued"></div>
             <input type="file">
             <button type="submit">Save</button>
-            {# <div class="comment-clear">Clear this comment</div> #}
```

Comment:
Yeah I prefer the deletion from comment index, it looks cleaner on the mocks as well.